test/functional/wallet-accounts.py

```diff
@@ -27,74 +29,74 @@ def __init__(self):
         self.node_args = [[]]
 
     def setup_network(self):
-        self.nodes = start_nodes(self.num_nodes, self.options.tmpdir, self.node_args)
+        self.nodes = start_nodes(self.num_nodes, self.options.tmpdir,
+                                 self.node_args)
```

Comment:
nit: As mentioned [here](https://github.com/bitcoin/bitcoin/pull/10224#discussion_r112836531), I don't think it makes sense to blindly apply pep8 to all python code. Also, changing white space in unrelated commits makes them harder to cherry-pick to other branches.